src/core/include/openvino/op/interpolate.hpp

```diff
@@ -73,6 +73,11 @@ class OPENVINO_API Interpolate : public Op {
 
 private:
     Attributes m_attrs;
+    template <class T>
+    friend void shape_infer(const Interpolate* op,
```

Comment:
Could you please move it from friend -- as we only use `m_attrs` here, which could be accessed via `get_attrs` method and make constant_data optional

Reply:
Yes, agree, removed.